src/xcode/ENA/ENA/Source/Views/GradientView.swift

```diff
@@ -61,6 +61,7 @@ class GradientView: UIView {
 		case green
 		case whiteWithGreyBorder
 		case whiteToLightBlue
+		case bottomLightBlueToTopWhite
```

Comment:
Just wondering if we remove Bottom and top here, as we have not used the orientations in enums above

Reply:
Just lightBlueToWhite?

Reply:
sounds good